package.json

```diff
@@ -39,6 +39,7 @@
     "eslint-config-prettier": "^9.1.0",
     "eslint-define-config": "^2.1.0",
     "eslint-plugin-prettier": "^5.2.1",
+    "glob": "^11.0.0",
```

Comment:
is this needed also in a root dependencies? it seems in `packages/extester` the glob is already added https://github.com/redhat-developer/vscode-extension-tester/blob/main/packages/extester/package.json#L69